.vscode/settings.json

```diff
@@ -1,11 +1,3 @@
 {
-  "ruby.useBundler": true,
-  "ruby.useLanguageServer": true,
-  "ruby.lint": {
-    "rubocop": {
-      "useBundler": true
-    }
-  },
-  "ruby.format": "rubocop",
-  "ruby.intellisense": "rubyLocate"
```

Comment:
- `.useBundler` was added in [PR #3923](https://github.com/forem/forem/pull/3923) to ensure ruby tools were executed with `bundle`. `rubyLsp` doesn't have [an equivalent setting](https://github.com/Shopify/ruby-lsp/blob/main/vscode/package.json), but it sounds like `rubyLsp` already "bundles" Ruby by default. from [this doc](https://marketplace.visualstudio.com/items?itemName=Shopify.ruby-lsp):

> To boot the server properly, the Ruby LSP uses a version manager to activate the right environment variables that point Bundler to the Ruby and gem paths.

- `.useLanguageServer` was also added in [PR #3923](https://github.com/forem/forem/pull/3923). this setting made sure the built-in language server was used, see this old [README](https://github.com/rubyide/vscode-ruby/tree/main/packages/vscode-ruby-client#using-the-language-server). `rubyLsp` doesn't have [an equivalent setting](https://github.com/Shopify/ruby-lsp/blob/main/vscode/package.json) because it doesn't need one -- it is **already** an implementation of the language server protocol for ruby. (`lsp` == language server protocol)

- `.lint` and `.format` both make sure rubocop is configured properly. this is now configured with `rubyLsp.formatter` 

- `.intellisense` - i couldn't find a `rubyLsp` [equivalent](https://github.com/Shopify/ruby-lsp/blob/main/vscode/package.json)